public/app/plugins/datasource/influxdb/datasource.ts

```diff
@@ -321,6 +321,19 @@ export default class InfluxDatasource extends DataSourceWithBackend<InfluxQuery,
   }
 
   targetContainsTemplate(target: any) {
+    if (this.isFlux) {
```

Comment:
Could we combine both ifs? The logic looks exactly the same.

Reply:
hmmm.. you are correct, it is the same. on the other hand, it is easier to think about all the cases when i split it like this, so if it's not a big problem, i'd prefer to leave them separate.